ci/docker/ml.build.Dockerfile

```diff
@@ -0,0 +1,16 @@
+ARG DOCKER_IMAGE_BASE_BUILD=cr.ray.io/rayproject/oss-ci-base_ml
+FROM $DOCKER_IMAGE_BASE_BUILD
+
+# Unset dind settings; we are using the host's docker daemon.
+ENV DOCKER_TLS_CERTDIR=
+ENV DOCKER_HOST=
+ENV DOCKER_TLS_VERIFY=
+ENV DOCKER_CERT_PATH=
+
+SHELL ["/bin/bash", "-ice"]
+
+COPY . .
+
+RUN pip install "mosaicml==0.12.1"
```

Comment:
could you preserve Kai's TODO tag to move this into requirements.txt ? and assign it to yourself maybe?